components/TodoList.js

```diff
@@ -1,37 +1,43 @@
-import  { useState } from 'react';
+import { useState } from 'react';
 
 function TodoList() {
-  const [todos, setTodos] = useState([]);  
-  const [newTodo, setNewTodo] = useState('');  
+  const [todos, setTodos] = useState([]);
+  const [newTodo, setNewTodo] = useState('');
 
-  function handleAddTodo() {
+  function addNewTodo() {
     if (newTodo.trim() !== '') {
       
-      setTodos([...todos, newTodo]);
+      const todoItem = {
+        id: Date.now(), 
```

Comment:
Take this instead =)
```suggestion
        id: Math.random().toString(32).substring(2), 
```